### DISCUSSION TITLE
can i submited the project after the dead line ?

### DISCUSSION THREAD

**Discussion by Maansimourya** [github/india#754] [GitHub Platform]
bcoz of my  exams  i couldnot  completed  my  learning paths and  now the  project deadline is given , can  i  submit my project  after the deadline ?

**Comment by Impavan007** [comment]
I want to know the same is it possible because my GitHub student devely account is not activated till now

**Reply by Impavan007** [reply]
It requires an edu acount

**Comment by ey-mentor1** [comment]
No project cannot be submitted after deadline.

**Comment by FRTMentor** [comment]
@Maansimourya Yes you cannot submit the project once after the given Deadline.
But there are some exceptional cases form the support team can able to extend the deadline with valid reasons.
Please reach out our support team.
support@futurereadytalent.in
